### ISSUE TITLE
Generated bindings cannot be compiled

### ISSUE THREAD

**Issue by szubtsovskiy** [Taaitaaiger/JlrsReflect.jl#1]
Hello @Taaitaaiger,

I just tried `JlrsReflect`, and it worked, but produced a struct which cannot be compiled because `JuliaString` does not derive `Debug` trait.

Given the Julia struct:
```julia
struct HistoricalReturnsInput
  country::String
  assets::Vector{String}
end
```

When I run

```julia
bindings = JlrsReflect.reflect([HistoricalReturnsInput]);

open("src/julia_bindings.rs", "w") do f
    write(f, bindings)
end
```

Then this Rust structure is generated:

```rust
#[repr(C)]
#[jlrs(julia_type = "Main.HistoricalReturnsInput")]
#[derive(Copy, Clone, Debug, JuliaStruct)]
pub struct HistoricalReturnsInput<'frame, 'data> {
    pub country: ::jlrs::value::string::JuliaString<'frame>,
    pub assets: ::jlrs::value::array::Array<'frame, 'data>,
}
```

And I am getting this compiler error:

```
error[E0277]: `JuliaString<'_>` doesn't implement `Debug`
 --> src/julia_bindings.rs:7:5
  |
7 |     pub country: ::jlrs::value::string::JuliaString<'frame>,
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `JuliaString<'_>` cannot be formatted using `{:?}` because it doesn't implement `Debug`
  |
  = help: the trait `Debug` is not implemented for `JuliaString<'_>`
  = note: required because of the requirements on the impl of `Debug` for `&JuliaString<'_>`
  = note: required for the cast to the object type `dyn Debug`
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

I guess, this should be addressed in `jlrs`, yet I am reporting it here, because it relates to the code generated by `JlrsReflect`. I am using `jlrs` 0.8.

**Comment by Taaitaaiger** [comment]
Ah right, I see that I've forgotten to add that for some of the builtin types. I might take the time to fix it and release 0.8.2 later today, otherwise that'll be tomorrow evening (CET). Thanks for the report!

**Comment by szubtsovskiy** [comment]
No rush @Taaitaaiger, I mostly wanted to let you know about it.

**Comment by Taaitaaiger** [comment]
No worries, it's one of those things that can be added quite easily. I've just released v0.8.2 which adds the missing debug implementations for builtin types.

**Comment by szubtsovskiy** [comment]
Great, thank you!
